2021-07-14 11:15:56,728 :: DEBUG :: > jishaku loaded!
2021-07-14 11:15:56,738 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:15:56,738 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:15:56,739 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:15:56,740 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:15:59,312 :: INFO :: -----------
2021-07-14 11:15:59,312 :: INFO :: The bot is ready.
2021-07-14 11:15:59,312 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:15:59,312 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:15:59,312 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:15:59,313 :: INFO :: 5 cogs are loaded
2021-07-14 11:15:59,313 :: INFO :: -----------
2021-07-14 11:15:59,313 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:16:52,264 :: DEBUG :: > jishaku loaded!
2021-07-14 11:16:52,273 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:16:52,274 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:16:52,274 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:16:52,275 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:16:54,983 :: INFO :: -----------
2021-07-14 11:16:54,983 :: INFO :: The bot is ready.
2021-07-14 11:16:54,983 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:16:54,983 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:16:54,983 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:16:54,983 :: INFO :: 5 cogs are loaded
2021-07-14 11:16:54,983 :: INFO :: -----------
2021-07-14 11:16:54,983 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:17:20,908 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -help
2021-07-14 11:28:37,855 :: DEBUG :: > jishaku loaded!
2021-07-14 11:28:37,864 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:28:37,865 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:28:37,865 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:28:37,866 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:28:40,501 :: INFO :: -----------
2021-07-14 11:28:40,501 :: INFO :: The bot is ready.
2021-07-14 11:28:40,501 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:28:40,501 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:28:40,501 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:28:40,501 :: INFO :: 5 cogs are loaded
2021-07-14 11:28:40,501 :: INFO :: -----------
2021-07-14 11:28:40,501 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:30:24,958 :: DEBUG :: > jishaku loaded!
2021-07-14 11:30:24,967 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:30:24,967 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:30:24,967 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:30:24,968 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:30:27,619 :: INFO :: -----------
2021-07-14 11:30:27,619 :: INFO :: The bot is ready.
2021-07-14 11:30:27,619 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:30:27,619 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:30:27,619 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:30:27,619 :: INFO :: 5 cogs are loaded
2021-07-14 11:30:27,619 :: INFO :: -----------
2021-07-14 11:30:27,620 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:30:43,332 :: DEBUG :: > jishaku loaded!
2021-07-14 11:30:43,342 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:30:43,342 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:30:43,343 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:30:43,343 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:30:43,344 :: ERROR :: > Failed to load extension cogs.utility
ExtensionFailed: Extension 'cogs.utility' raised an error: ModuleNotFoundError: No module named 'flask'
Traceback (most recent call last):
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/utility.py", line 3, in <module>
    import flask
ModuleNotFoundError: No module named 'flask'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.utility' raised an error: ModuleNotFoundError: No module named 'flask'
2021-07-14 11:30:45,959 :: INFO :: -----------
2021-07-14 11:30:45,959 :: INFO :: The bot is ready.
2021-07-14 11:30:45,959 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:30:45,959 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:30:45,959 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:30:45,959 :: INFO :: 5 cogs are loaded
2021-07-14 11:30:45,959 :: INFO :: -----------
2021-07-14 11:30:45,959 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:31:00,202 :: DEBUG :: > jishaku loaded!
2021-07-14 11:31:00,211 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:31:00,211 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:31:00,212 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:31:00,212 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:31:00,261 :: ERROR :: > Failed to load extension cogs.utility
ExtensionFailed: Extension 'cogs.utility' raised an error: NameError: name 'Flask' is not defined
Traceback (most recent call last):
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/utility.py", line 9, in <module>
    app = Flask(__name__)
NameError: name 'Flask' is not defined

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.utility' raised an error: NameError: name 'Flask' is not defined
2021-07-14 11:31:02,878 :: INFO :: -----------
2021-07-14 11:31:02,878 :: INFO :: The bot is ready.
2021-07-14 11:31:02,878 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:31:02,878 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:31:02,878 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:31:02,878 :: INFO :: 5 cogs are loaded
2021-07-14 11:31:02,878 :: INFO :: -----------
2021-07-14 11:31:02,878 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:31:15,098 :: DEBUG :: > jishaku loaded!
2021-07-14 11:31:15,108 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:31:15,108 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:31:15,108 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:31:15,109 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:31:15,109 :: ERROR :: > Failed to load extension cogs.utility
ExtensionFailed: Extension 'cogs.utility' raised an error: SyntaxError: invalid syntax (utility.py, line 3)
Traceback (most recent call last):
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 844, in exec_module
  File "<frozen importlib._bootstrap_external>", line 981, in get_code
  File "<frozen importlib._bootstrap_external>", line 911, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/utility.py", line 3
    import flask import Flask
                 ^
SyntaxError: invalid syntax

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.utility' raised an error: SyntaxError: invalid syntax (utility.py, line 3)
2021-07-14 11:31:17,694 :: INFO :: -----------
2021-07-14 11:31:17,694 :: INFO :: The bot is ready.
2021-07-14 11:31:17,694 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:31:17,695 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:31:17,695 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:31:17,695 :: INFO :: 5 cogs are loaded
2021-07-14 11:31:17,695 :: INFO :: -----------
2021-07-14 11:31:17,695 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:31:24,751 :: DEBUG :: > jishaku loaded!
2021-07-14 11:31:24,760 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:31:24,760 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:31:24,760 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:31:24,761 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:31:24,811 :: ERROR :: > Failed to load extension cogs.utility
NoEntryPointError: Extension 'cogs.utility' has no 'setup' function.
Traceback (most recent call last):
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 612, in _load_from_module_spec
    setup = getattr(lib, 'setup')
AttributeError: module 'cogs.utility' has no attribute 'setup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/venv/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 615, in _load_from_module_spec
    raise errors.NoEntryPointError(key)
discord.ext.commands.errors.NoEntryPointError: Extension 'cogs.utility' has no 'setup' function.
2021-07-14 11:31:27,468 :: INFO :: -----------
2021-07-14 11:31:27,468 :: INFO :: The bot is ready.
2021-07-14 11:31:27,468 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:31:27,468 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:31:27,468 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:31:27,468 :: INFO :: 5 cogs are loaded
2021-07-14 11:31:27,468 :: INFO :: -----------
2021-07-14 11:31:27,468 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:31:37,526 :: DEBUG :: > jishaku loaded!
2021-07-14 11:31:37,535 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:31:37,535 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:31:37,535 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:31:37,536 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:31:37,588 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:31:40,215 :: INFO :: -----------
2021-07-14 11:31:40,215 :: INFO :: The bot is ready.
2021-07-14 11:31:40,215 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:31:40,215 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:31:40,215 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:31:40,216 :: INFO :: 6 cogs are loaded
2021-07-14 11:31:40,216 :: INFO :: -----------
2021-07-14 11:36:20,566 :: DEBUG :: > jishaku loaded!
2021-07-14 11:36:20,578 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:36:20,579 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:36:20,580 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:36:20,580 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:36:20,581 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:36:20,594 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:36:23,232 :: INFO :: -----------
2021-07-14 11:36:23,232 :: INFO :: The bot is ready.
2021-07-14 11:36:23,232 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:36:23,232 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:36:23,233 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:36:23,233 :: INFO :: 7 cogs are loaded
2021-07-14 11:36:23,233 :: INFO :: -----------
2021-07-14 11:36:23,233 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:36:23,234 :: INFO :: dOGbot JSON API listening
2021-07-14 11:36:27,326 :: INFO :: 127.0.0.1 [14/Jul/2021:16:36:27 +0000] "GET / HTTP/1.1" 200 164 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
2021-07-14 11:37:14,250 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:37:15,377 :: DEBUG :: > jishaku loaded!
2021-07-14 11:37:15,387 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:37:15,387 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:37:15,387 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:37:15,388 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:37:15,388 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:37:15,399 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:37:18,175 :: INFO :: -----------
2021-07-14 11:37:18,175 :: INFO :: The bot is ready.
2021-07-14 11:37:18,175 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:37:18,175 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:37:18,175 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:37:18,175 :: INFO :: 7 cogs are loaded
2021-07-14 11:37:18,176 :: INFO :: -----------
2021-07-14 11:37:18,176 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:37:36,186 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:37:37,300 :: DEBUG :: > jishaku loaded!
2021-07-14 11:37:37,310 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:37:37,310 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:37:37,311 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:37:37,311 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:37:37,311 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:37:37,323 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:37:39,980 :: INFO :: -----------
2021-07-14 11:37:39,981 :: INFO :: The bot is ready.
2021-07-14 11:37:39,981 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:37:39,981 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:37:39,981 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:37:39,981 :: INFO :: 7 cogs are loaded
2021-07-14 11:37:39,981 :: INFO :: -----------
2021-07-14 11:37:39,981 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:38:51,816 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:38:52,948 :: DEBUG :: > jishaku loaded!
2021-07-14 11:38:52,958 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:38:52,958 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:38:52,959 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:38:52,960 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:38:52,960 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:38:52,973 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:38:55,647 :: INFO :: -----------
2021-07-14 11:38:55,647 :: INFO :: The bot is ready.
2021-07-14 11:38:55,647 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:38:55,647 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:38:55,647 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:38:55,647 :: INFO :: 7 cogs are loaded
2021-07-14 11:38:55,647 :: INFO :: -----------
2021-07-14 11:38:55,648 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:39:09,329 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:39:10,356 :: DEBUG :: > jishaku loaded!
2021-07-14 11:39:10,367 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:39:10,368 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:39:10,368 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:39:10,369 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:39:10,369 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:39:10,380 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:39:13,004 :: INFO :: -----------
2021-07-14 11:39:13,004 :: INFO :: The bot is ready.
2021-07-14 11:39:13,004 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:39:13,004 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:39:13,004 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:39:13,004 :: INFO :: 7 cogs are loaded
2021-07-14 11:39:13,004 :: INFO :: -----------
2021-07-14 11:39:13,005 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:39:13,005 :: INFO :: dOGbot JSON API listening
2021-07-14 11:39:16,643 :: INFO :: 127.0.0.1 [14/Jul/2021:16:39:16 +0000] "GET / HTTP/1.1" 200 173 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
2021-07-14 11:54:13,036 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:54:25,855 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:54:47,746 :: DEBUG :: > jishaku loaded!
2021-07-14 11:54:47,755 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:54:47,756 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:54:47,756 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:54:47,757 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:54:47,758 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:54:47,770 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:54:47,771 :: ERROR :: > Failed to load extension cogs.fun
ExtensionFailed: Extension 'cogs.fun' raised an error: TypeError: setup() missing 1 required positional argument: 'bot'
Traceback (most recent call last):
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 605, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/fun.py", line 18, in <module>
    setup = Fun.setup()
TypeError: setup() missing 1 required positional argument: 'bot'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 677, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 608, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.fun' raised an error: TypeError: setup() missing 1 required positional argument: 'bot'
2021-07-14 11:54:50,396 :: INFO :: -----------
2021-07-14 11:54:50,397 :: INFO :: The bot is ready.
2021-07-14 11:54:50,397 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:54:50,397 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:54:50,397 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:54:50,397 :: INFO :: 7 cogs are loaded
2021-07-14 11:54:50,397 :: INFO :: -----------
2021-07-14 11:54:50,397 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:54:50,398 :: INFO :: dOGbot JSON API listening
2021-07-14 11:55:13,909 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:55:15,080 :: DEBUG :: > jishaku loaded!
2021-07-14 11:55:15,090 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:55:15,090 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:55:15,091 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:55:15,091 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:55:15,092 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:55:15,103 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:55:15,104 :: DEBUG :: > cogs.fun loaded!
2021-07-14 11:55:17,708 :: INFO :: -----------
2021-07-14 11:55:17,708 :: INFO :: The bot is ready.
2021-07-14 11:55:17,708 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:55:17,708 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:55:17,708 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:55:17,708 :: INFO :: 8 cogs are loaded
2021-07-14 11:55:17,709 :: INFO :: -----------
2021-07-14 11:55:17,709 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:55:17,709 :: INFO :: dOGbot JSON API listening
2021-07-14 11:55:24,891 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -p
2021-07-14 11:55:26,919 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -p
2021-07-14 11:55:28,151 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -p
2021-07-14 11:55:29,059 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -p
2021-07-14 11:55:32,112 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -kuki
2021-07-14 11:56:20,213 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:56:21,511 :: DEBUG :: > jishaku loaded!
2021-07-14 11:56:21,521 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:56:21,521 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:56:21,522 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:56:21,523 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:56:21,523 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:56:21,534 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:56:21,534 :: DEBUG :: > cogs.fun loaded!
2021-07-14 11:56:24,056 :: INFO :: -----------
2021-07-14 11:56:24,057 :: INFO :: The bot is ready.
2021-07-14 11:56:24,057 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:56:24,057 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:56:24,057 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:56:24,057 :: INFO :: 8 cogs are loaded
2021-07-14 11:56:24,057 :: INFO :: -----------
2021-07-14 11:56:24,057 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:56:24,058 :: INFO :: dOGbot JSON API listening
2021-07-14 11:56:30,035 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -help
2021-07-14 11:57:18,498 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 11:57:19,758 :: DEBUG :: > jishaku loaded!
2021-07-14 11:57:19,767 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 11:57:19,768 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 11:57:19,768 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 11:57:19,769 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 11:57:19,769 :: DEBUG :: > cogs.utility loaded!
2021-07-14 11:57:19,780 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 11:57:19,780 :: DEBUG :: > cogs.fun loaded!
2021-07-14 11:57:22,304 :: INFO :: -----------
2021-07-14 11:57:22,305 :: INFO :: The bot is ready.
2021-07-14 11:57:22,305 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 11:57:22,305 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 11:57:22,305 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 11:57:22,305 :: INFO :: 8 cogs are loaded
2021-07-14 11:57:22,305 :: INFO :: -----------
2021-07-14 11:57:22,305 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 11:57:22,306 :: INFO :: dOGbot JSON API listening
2021-07-14 11:57:23,361 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -help
2021-07-14 11:58:06,149 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -shards
2021-07-14 12:00:34,695 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 12:00:35,511 :: DEBUG :: > jishaku loaded!
2021-07-14 12:00:35,520 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 12:00:35,520 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 12:00:35,521 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 12:00:35,522 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 12:00:35,522 :: DEBUG :: > cogs.utility loaded!
2021-07-14 12:00:35,533 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 12:00:35,534 :: DEBUG :: > cogs.fun loaded!
2021-07-14 12:00:38,096 :: INFO :: -----------
2021-07-14 12:00:38,096 :: INFO :: The bot is ready.
2021-07-14 12:00:38,096 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 12:00:38,096 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 12:00:38,097 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 12:00:38,097 :: INFO :: 8 cogs are loaded
2021-07-14 12:00:38,097 :: INFO :: -----------
2021-07-14 12:00:38,097 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 12:00:38,098 :: INFO :: dOGbot JSON API listening
2021-07-14 12:00:39,476 :: INFO :: 778757191770308628 - #general :: <hydrostaticcog#2330> -help
2021-07-14 12:15:38,130 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 12:16:11,582 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 12:17:56,745 :: DEBUG :: > jishaku loaded!
2021-07-14 12:17:56,754 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 12:17:56,754 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 12:17:56,755 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 12:17:56,755 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 12:17:56,755 :: DEBUG :: > cogs.utility loaded!
2021-07-14 12:17:56,766 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 12:17:56,766 :: DEBUG :: > cogs.fun loaded!
2021-07-14 12:17:59,451 :: INFO :: -----------
2021-07-14 12:17:59,451 :: INFO :: The bot is ready.
2021-07-14 12:17:59,451 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 12:17:59,451 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 12:17:59,451 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 12:17:59,451 :: INFO :: 8 cogs are loaded
2021-07-14 12:17:59,451 :: INFO :: -----------
2021-07-14 12:17:59,451 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 12:17:59,452 :: INFO :: dOGbot JSON API listening
2021-07-14 12:18:06,224 :: INFO :: dOGbot JSON API is shutting down...
2021-07-14 12:18:07,652 :: DEBUG :: > jishaku loaded!
2021-07-14 12:18:07,661 :: DEBUG :: > cogs.error_handling loaded!
2021-07-14 12:18:07,661 :: DEBUG :: > cogs.background_loop loaded!
2021-07-14 12:18:07,662 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-14 12:18:07,662 :: DEBUG :: > cogs.database_commands loaded!
2021-07-14 12:18:07,663 :: DEBUG :: > cogs.utility loaded!
2021-07-14 12:18:07,673 :: DEBUG :: > cogs.rest_api loaded!
2021-07-14 12:18:07,673 :: DEBUG :: > cogs.fun loaded!
2021-07-14 12:18:10,464 :: INFO :: -----------
2021-07-14 12:18:10,464 :: INFO :: The bot is ready.
2021-07-14 12:18:10,464 :: INFO :: Logged in as testbot (780811646981111848).
2021-07-14 12:18:10,464 :: INFO :: I see 1 guilds, and 1 members.
2021-07-14 12:18:10,464 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=780811646981111848&scope=bot&permissions=0
2021-07-14 12:18:10,464 :: INFO :: 8 cogs are loaded
2021-07-14 12:18:10,464 :: INFO :: -----------
2021-07-14 12:18:10,464 :: DEBUG :: 778757191770308628 - #test :: Updating status message
2021-07-14 12:18:10,465 :: INFO :: dOGbot JSON API listening
2021-07-14 12:22:28,036 :: INFO :: 127.0.0.1 [14/Jul/2021:17:22:28 +0000] "GET / HTTP/1.1" 200 173 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
2021-07-14 12:26:23,755 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:19:36,855 :: DEBUG :: > jishaku loaded!
2021-07-19 21:19:36,866 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:19:36,866 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:19:36,867 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:19:36,867 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:19:36,868 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:19:36,868 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-07-19 21:19:36,869 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:19:39,443 :: INFO :: -----------
2021-07-19 21:19:39,443 :: INFO :: The bot is ready.
2021-07-19 21:19:39,443 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:19:39,443 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:19:39,443 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:19:39,443 :: INFO :: 7 cogs are loaded
2021-07-19 21:19:39,443 :: INFO :: -----------
2021-07-19 21:19:39,443 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:24:45,356 :: DEBUG :: > jishaku loaded!
2021-07-19 21:24:45,365 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:24:45,366 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:24:45,366 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:24:45,367 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:24:45,367 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:24:45,367 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/media/hydrostaticcog/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydrostaticcog/.local/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-07-19 21:24:45,368 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:24:47,921 :: INFO :: -----------
2021-07-19 21:24:47,921 :: INFO :: The bot is ready.
2021-07-19 21:24:47,921 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:24:47,921 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:24:47,921 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:24:47,921 :: INFO :: 7 cogs are loaded
2021-07-19 21:24:47,922 :: INFO :: -----------
2021-07-19 21:24:47,922 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:25:01,591 :: DEBUG :: > jishaku loaded!
2021-07-19 21:25:01,600 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:25:01,600 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:25:01,600 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:25:01,601 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:25:01,601 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:25:01,611 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:25:01,612 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:25:04,150 :: INFO :: -----------
2021-07-19 21:25:04,150 :: INFO :: The bot is ready.
2021-07-19 21:25:04,150 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:25:04,150 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:25:04,150 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:25:04,150 :: INFO :: 8 cogs are loaded
2021-07-19 21:25:04,150 :: INFO :: -----------
2021-07-19 21:25:04,150 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:25:04,151 :: INFO :: dOGbot JSON API listening
2021-07-19 21:25:09,800 :: INFO :: 772889538300608512 - #developers-💻 :: <hydrostaticcog#2330> -jsk
2021-07-19 21:25:19,231 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:29:00,569 :: DEBUG :: > jishaku loaded!
2021-07-19 21:29:00,577 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:29:00,578 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:29:00,578 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:29:00,579 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:29:00,579 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:29:00,589 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:29:00,590 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:29:03,270 :: INFO :: -----------
2021-07-19 21:29:03,270 :: INFO :: The bot is ready.
2021-07-19 21:29:03,270 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:29:03,270 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:29:03,270 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:29:03,270 :: INFO :: 8 cogs are loaded
2021-07-19 21:29:03,270 :: INFO :: -----------
2021-07-19 21:29:03,271 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:29:03,271 :: INFO :: dOGbot JSON API listening
2021-07-19 21:29:56,006 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:29:57,004 :: DEBUG :: > jishaku loaded!
2021-07-19 21:29:57,012 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:29:57,013 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:29:57,013 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:29:57,014 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:29:57,014 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:29:57,025 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:29:57,026 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:29:59,675 :: INFO :: -----------
2021-07-19 21:29:59,675 :: INFO :: The bot is ready.
2021-07-19 21:29:59,675 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:29:59,675 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:29:59,675 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:29:59,675 :: INFO :: 8 cogs are loaded
2021-07-19 21:29:59,675 :: INFO :: -----------
2021-07-19 21:29:59,676 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:29:59,676 :: INFO :: dOGbot JSON API listening
2021-07-19 21:32:40,846 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:32:41,880 :: DEBUG :: > jishaku loaded!
2021-07-19 21:32:41,888 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:32:41,889 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:32:41,889 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:32:41,889 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:32:41,890 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:32:41,901 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:32:41,901 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:32:44,450 :: INFO :: -----------
2021-07-19 21:32:44,450 :: INFO :: The bot is ready.
2021-07-19 21:32:44,450 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:32:44,450 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:32:44,451 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:32:44,451 :: INFO :: 8 cogs are loaded
2021-07-19 21:32:44,451 :: INFO :: -----------
2021-07-19 21:32:44,451 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:32:44,451 :: INFO :: dOGbot JSON API listening
2021-07-19 21:32:59,989 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:33:01,111 :: DEBUG :: > jishaku loaded!
2021-07-19 21:33:01,121 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:33:01,122 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:33:01,122 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:33:01,123 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:33:01,123 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:33:01,134 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:33:01,135 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:33:03,766 :: INFO :: -----------
2021-07-19 21:33:03,766 :: INFO :: The bot is ready.
2021-07-19 21:33:03,766 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:33:03,766 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:33:03,766 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:33:03,766 :: INFO :: 8 cogs are loaded
2021-07-19 21:33:03,766 :: INFO :: -----------
2021-07-19 21:33:03,766 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:33:03,767 :: INFO :: dOGbot JSON API listening
2021-07-19 21:34:07,485 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:34:08,483 :: DEBUG :: > jishaku loaded!
2021-07-19 21:34:08,491 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:34:08,492 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:34:08,492 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:34:08,493 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:34:08,493 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:34:08,504 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:34:08,505 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:34:11,071 :: INFO :: -----------
2021-07-19 21:34:11,071 :: INFO :: The bot is ready.
2021-07-19 21:34:11,071 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:34:11,072 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:34:11,072 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:34:11,072 :: INFO :: 8 cogs are loaded
2021-07-19 21:34:11,072 :: INFO :: -----------
2021-07-19 21:34:11,072 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:34:11,072 :: INFO :: dOGbot JSON API listening
2021-07-19 21:34:54,537 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:34:55,469 :: DEBUG :: > jishaku loaded!
2021-07-19 21:34:55,477 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:34:55,478 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:34:55,478 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:34:55,479 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:34:55,479 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:34:55,492 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:34:55,492 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:34:58,038 :: INFO :: -----------
2021-07-19 21:34:58,038 :: INFO :: The bot is ready.
2021-07-19 21:34:58,038 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:34:58,038 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:34:58,039 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:34:58,039 :: INFO :: 8 cogs are loaded
2021-07-19 21:34:58,039 :: INFO :: -----------
2021-07-19 21:34:58,039 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:34:58,039 :: INFO :: dOGbot JSON API listening
2021-07-19 21:35:08,361 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:35:09,312 :: DEBUG :: > jishaku loaded!
2021-07-19 21:35:09,320 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:35:09,321 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:35:09,321 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:35:09,322 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:35:09,322 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:35:09,333 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:35:09,333 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:35:11,896 :: INFO :: -----------
2021-07-19 21:35:11,896 :: INFO :: The bot is ready.
2021-07-19 21:35:11,896 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:35:11,896 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:35:11,896 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:35:11,896 :: INFO :: 8 cogs are loaded
2021-07-19 21:35:11,896 :: INFO :: -----------
2021-07-19 21:35:11,896 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:35:11,896 :: INFO :: dOGbot JSON API listening
2021-07-19 21:36:24,139 :: INFO :: dOGbot JSON API is shutting down...
2021-07-19 21:36:24,958 :: DEBUG :: > jishaku loaded!
2021-07-19 21:36:24,966 :: DEBUG :: > cogs.error_handling loaded!
2021-07-19 21:36:24,967 :: DEBUG :: > cogs.background_loop loaded!
2021-07-19 21:36:24,967 :: DEBUG :: > cogs.support_server_commands loaded!
2021-07-19 21:36:24,968 :: DEBUG :: > cogs.database_commands loaded!
2021-07-19 21:36:24,968 :: DEBUG :: > cogs.utility loaded!
2021-07-19 21:36:24,979 :: DEBUG :: > cogs.rest_api loaded!
2021-07-19 21:36:24,979 :: DEBUG :: > cogs.fun loaded!
2021-07-19 21:36:27,647 :: INFO :: -----------
2021-07-19 21:36:27,647 :: INFO :: The bot is ready.
2021-07-19 21:36:27,647 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-07-19 21:36:27,647 :: INFO :: I see 3 guilds, and 1 members.
2021-07-19 21:36:27,647 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-07-19 21:36:27,648 :: INFO :: 8 cogs are loaded
2021-07-19 21:36:27,648 :: INFO :: -----------
2021-07-19 21:36:27,648 :: WARNING :: The status channel for the support server command is configured improperly.
2021-07-19 21:36:27,648 :: INFO :: dOGbot JSON API listening
2021-07-19 21:39:13,890 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:43:41,339 :: DEBUG :: > jishaku loaded!
2021-10-19 14:43:41,349 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:43:41,350 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:43:41,351 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:43:41,352 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:43:41,352 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:43:41,353 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:43:41,354 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:43:44,082 :: INFO :: -----------
2021-10-19 14:43:44,082 :: INFO :: The bot is ready.
2021-10-19 14:43:44,082 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:43:44,082 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:43:44,082 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:43:44,082 :: INFO :: 7 cogs are loaded
2021-10-19 14:43:44,082 :: INFO :: -----------
2021-10-19 14:43:44,082 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:44:55,542 :: DEBUG :: > jishaku loaded!
2021-10-19 14:44:55,552 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:44:55,552 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:44:55,553 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:44:55,553 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:44:55,554 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:44:55,554 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:44:55,554 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:44:58,193 :: INFO :: -----------
2021-10-19 14:44:58,193 :: INFO :: The bot is ready.
2021-10-19 14:44:58,193 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:44:58,194 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:44:58,194 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:44:58,194 :: INFO :: 7 cogs are loaded
2021-10-19 14:44:58,194 :: INFO :: -----------
2021-10-19 14:44:58,194 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:46:14,839 :: DEBUG :: > jishaku loaded!
2021-10-19 14:46:14,849 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:46:14,849 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:46:14,849 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:46:14,850 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:46:14,850 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:46:14,850 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:46:14,851 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:46:17,636 :: INFO :: -----------
2021-10-19 14:46:17,636 :: INFO :: The bot is ready.
2021-10-19 14:46:17,637 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:46:17,637 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:46:17,637 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:46:17,637 :: INFO :: 7 cogs are loaded
2021-10-19 14:46:17,637 :: INFO :: -----------
2021-10-19 14:46:17,637 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:46:41,510 :: DEBUG :: > jishaku loaded!
2021-10-19 14:46:41,519 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:46:41,520 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:46:41,520 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:46:41,520 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:46:41,521 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:46:41,521 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:46:41,522 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:46:44,109 :: INFO :: -----------
2021-10-19 14:46:44,110 :: INFO :: The bot is ready.
2021-10-19 14:46:44,110 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:46:44,110 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:46:44,110 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:46:44,110 :: INFO :: 7 cogs are loaded
2021-10-19 14:46:44,110 :: INFO :: -----------
2021-10-19 14:46:44,110 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:48:40,743 :: DEBUG :: > jishaku loaded!
2021-10-19 14:48:40,752 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:48:40,753 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:48:40,753 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:48:40,754 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:48:40,754 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:48:40,754 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:48:40,755 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:48:43,415 :: INFO :: -----------
2021-10-19 14:48:43,415 :: INFO :: The bot is ready.
2021-10-19 14:48:43,416 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:48:43,416 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:48:43,416 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:48:43,416 :: INFO :: 7 cogs are loaded
2021-10-19 14:48:43,416 :: INFO :: -----------
2021-10-19 14:48:43,416 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:48:56,071 :: DEBUG :: > jishaku loaded!
2021-10-19 14:48:56,081 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:48:56,081 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:48:56,082 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:48:56,082 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:48:56,082 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:48:56,083 :: ERROR :: > Failed to load extension cogs.rest_api
ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/rest_api.py", line 7, in <module>
    import aiohttp_cors
ModuleNotFoundError: No module named 'aiohttp_cors'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/main.py", line 46, in <module>
    bot.load_extension(cog_name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'cogs.rest_api' raised an error: ModuleNotFoundError: No module named 'aiohttp_cors'
2021-10-19 14:48:56,083 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:48:58,569 :: INFO :: -----------
2021-10-19 14:48:58,570 :: INFO :: The bot is ready.
2021-10-19 14:48:58,570 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:48:58,570 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:48:58,570 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:48:58,570 :: INFO :: 7 cogs are loaded
2021-10-19 14:48:58,570 :: INFO :: -----------
2021-10-19 14:48:58,570 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:53:28,313 :: DEBUG :: > jishaku loaded!
2021-10-19 14:53:28,322 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:53:28,323 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:53:28,323 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:53:28,323 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:53:28,324 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:53:28,340 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:53:28,340 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:53:31,063 :: INFO :: -----------
2021-10-19 14:53:31,063 :: INFO :: The bot is ready.
2021-10-19 14:53:31,064 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:53:31,064 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:53:31,064 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:53:31,064 :: INFO :: 8 cogs are loaded
2021-10-19 14:53:31,064 :: INFO :: -----------
2021-10-19 14:53:31,064 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:53:31,064 :: INFO :: dOGbot JSON API listening
2021-10-19 14:53:40,375 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitter
2021-10-19 14:54:12,043 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:54:12,560 :: DEBUG :: > jishaku loaded!
2021-10-19 14:54:12,571 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:54:12,571 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:54:12,571 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:54:12,572 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:54:12,573 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:54:12,584 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:54:12,584 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:54:15,294 :: INFO :: -----------
2021-10-19 14:54:15,294 :: INFO :: The bot is ready.
2021-10-19 14:54:15,294 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:54:15,294 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:54:15,294 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:54:15,294 :: INFO :: 8 cogs are loaded
2021-10-19 14:54:15,294 :: INFO :: -----------
2021-10-19 14:54:15,294 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:54:15,294 :: INFO :: dOGbot JSON API listening
2021-10-19 14:54:15,999 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -youtube
2021-10-19 14:54:38,022 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:54:38,677 :: DEBUG :: > jishaku loaded!
2021-10-19 14:54:38,687 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:54:38,688 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:54:38,688 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:54:38,689 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:54:38,689 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:54:38,700 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:54:38,701 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:54:41,367 :: INFO :: -----------
2021-10-19 14:54:41,368 :: INFO :: The bot is ready.
2021-10-19 14:54:41,368 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:54:41,368 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:54:41,368 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:54:41,368 :: INFO :: 8 cogs are loaded
2021-10-19 14:54:41,368 :: INFO :: -----------
2021-10-19 14:54:41,368 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:54:41,368 :: INFO :: dOGbot JSON API listening
2021-10-19 14:54:47,646 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitter
2021-10-19 14:54:53,422 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:54:54,104 :: DEBUG :: > jishaku loaded!
2021-10-19 14:54:54,113 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:54:54,114 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:54:54,114 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:54:54,114 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:54:54,115 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:54:54,126 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:54:54,126 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:54:56,703 :: INFO :: -----------
2021-10-19 14:54:56,703 :: INFO :: The bot is ready.
2021-10-19 14:54:56,703 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:54:56,703 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:54:56,703 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:54:56,703 :: INFO :: 8 cogs are loaded
2021-10-19 14:54:56,703 :: INFO :: -----------
2021-10-19 14:54:56,703 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:54:56,703 :: INFO :: dOGbot JSON API listening
2021-10-19 14:54:58,382 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitter
2021-10-19 14:55:57,950 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:55:58,472 :: DEBUG :: > jishaku loaded!
2021-10-19 14:55:58,481 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:55:58,482 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:55:58,482 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:55:58,483 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:55:58,483 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:55:58,494 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:55:58,494 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:56:01,142 :: INFO :: -----------
2021-10-19 14:56:01,142 :: INFO :: The bot is ready.
2021-10-19 14:56:01,142 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:56:01,142 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:56:01,142 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:56:01,142 :: INFO :: 8 cogs are loaded
2021-10-19 14:56:01,142 :: INFO :: -----------
2021-10-19 14:56:01,142 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:56:01,143 :: INFO :: dOGbot JSON API listening
2021-10-19 14:56:03,635 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitter
2021-10-19 14:56:49,234 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 14:56:49,739 :: DEBUG :: > jishaku loaded!
2021-10-19 14:56:49,749 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 14:56:49,749 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 14:56:49,749 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 14:56:49,751 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 14:56:49,752 :: DEBUG :: > cogs.utility loaded!
2021-10-19 14:56:49,764 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 14:56:49,764 :: DEBUG :: > cogs.fun loaded!
2021-10-19 14:56:52,372 :: INFO :: -----------
2021-10-19 14:56:52,372 :: INFO :: The bot is ready.
2021-10-19 14:56:52,373 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 14:56:52,373 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 14:56:52,373 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 14:56:52,373 :: INFO :: 8 cogs are loaded
2021-10-19 14:56:52,373 :: INFO :: -----------
2021-10-19 14:56:52,373 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 14:56:52,373 :: INFO :: dOGbot JSON API listening
2021-10-19 14:56:59,734 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitter
2021-10-19 14:57:06,858 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -youtube
2021-10-19 14:57:12,833 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -twitch
2021-10-19 14:57:26,053 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -kuki
2021-10-19 14:57:32,203 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:00:01,051 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:23:41,321 :: DEBUG :: > jishaku loaded!
2021-10-19 15:23:41,330 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:23:41,331 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:23:41,331 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:23:41,332 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:23:41,332 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:23:41,343 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:23:41,343 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:23:44,006 :: INFO :: -----------
2021-10-19 15:23:44,007 :: INFO :: The bot is ready.
2021-10-19 15:23:44,007 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:23:44,007 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:23:44,007 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:23:44,007 :: INFO :: 8 cogs are loaded
2021-10-19 15:23:44,007 :: INFO :: -----------
2021-10-19 15:23:44,007 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:23:44,007 :: INFO :: dOGbot JSON API listening
2021-10-19 15:26:01,448 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:26:01,983 :: DEBUG :: > jishaku loaded!
2021-10-19 15:26:01,993 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:26:01,994 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:26:01,994 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:26:01,994 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:26:01,995 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:26:02,006 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:26:02,006 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:26:04,631 :: INFO :: -----------
2021-10-19 15:26:04,631 :: INFO :: The bot is ready.
2021-10-19 15:26:04,631 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:26:04,631 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:26:04,631 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:26:04,631 :: INFO :: 8 cogs are loaded
2021-10-19 15:26:04,631 :: INFO :: -----------
2021-10-19 15:26:04,632 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:26:04,632 :: INFO :: dOGbot JSON API listening
2021-10-19 15:26:08,347 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:26:08,575 :: DEBUG :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> Error during processing: Command raised an exception: AttributeError: 'Embed' object has no attribute 'addField' (CommandInvokeError("Command raised an exception: AttributeError: 'Embed' object has no attribute 'addField'"))
2021-10-19 15:26:08,576 :: ERROR :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/run/media/hydro/D/Code/DSBOTS/dOGbot/cogs/utility.py", line 21, in ping
    embed.addField(name="API Ping", value=time_delta, inline=False)
AttributeError: 'Embed' object has no attribute 'addField'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/home/hydro/.local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'Embed' object has no attribute 'addField'

2021-10-19 15:26:57,981 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:26:58,504 :: DEBUG :: > jishaku loaded!
2021-10-19 15:26:58,514 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:26:58,514 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:26:58,514 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:26:58,515 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:26:58,516 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:26:58,526 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:26:58,527 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:27:01,022 :: INFO :: -----------
2021-10-19 15:27:01,022 :: INFO :: The bot is ready.
2021-10-19 15:27:01,023 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:27:01,023 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:27:01,023 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:27:01,023 :: INFO :: 8 cogs are loaded
2021-10-19 15:27:01,023 :: INFO :: -----------
2021-10-19 15:27:01,023 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:27:01,023 :: INFO :: dOGbot JSON API listening
2021-10-19 15:27:02,781 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:28:25,295 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:28:25,812 :: DEBUG :: > jishaku loaded!
2021-10-19 15:28:25,822 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:28:25,822 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:28:25,822 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:28:25,823 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:28:25,824 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:28:25,834 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:28:25,834 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:28:28,543 :: INFO :: -----------
2021-10-19 15:28:28,543 :: INFO :: The bot is ready.
2021-10-19 15:28:28,544 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:28:28,544 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:28:28,544 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:28:28,544 :: INFO :: 8 cogs are loaded
2021-10-19 15:28:28,544 :: INFO :: -----------
2021-10-19 15:28:28,544 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:28:28,544 :: INFO :: dOGbot JSON API listening
2021-10-19 15:28:30,257 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:29:05,606 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:29:06,115 :: DEBUG :: > jishaku loaded!
2021-10-19 15:29:06,124 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:29:06,125 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:29:06,125 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:29:06,126 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:29:06,126 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:29:06,137 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:29:06,137 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:29:08,742 :: INFO :: -----------
2021-10-19 15:29:08,743 :: INFO :: The bot is ready.
2021-10-19 15:29:08,743 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:29:08,743 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:29:08,743 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:29:08,743 :: INFO :: 8 cogs are loaded
2021-10-19 15:29:08,743 :: INFO :: -----------
2021-10-19 15:29:08,743 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:29:08,743 :: INFO :: dOGbot JSON API listening
2021-10-19 15:29:10,595 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:29:54,854 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:29:55,376 :: DEBUG :: > jishaku loaded!
2021-10-19 15:29:55,387 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:29:55,387 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:29:55,387 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:29:55,388 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:29:55,389 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:29:55,400 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:29:55,400 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:29:57,996 :: INFO :: -----------
2021-10-19 15:29:57,996 :: INFO :: The bot is ready.
2021-10-19 15:29:57,997 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:29:57,997 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:29:57,997 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:29:57,997 :: INFO :: 8 cogs are loaded
2021-10-19 15:29:57,997 :: INFO :: -----------
2021-10-19 15:29:57,997 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:29:57,997 :: INFO :: dOGbot JSON API listening
2021-10-19 15:29:59,698 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:30:40,743 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:30:41,233 :: DEBUG :: > jishaku loaded!
2021-10-19 15:30:41,242 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:30:41,243 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:30:41,243 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:30:41,244 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:30:41,244 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:30:41,255 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:30:41,255 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:30:43,872 :: INFO :: -----------
2021-10-19 15:30:43,872 :: INFO :: The bot is ready.
2021-10-19 15:30:43,872 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:30:43,873 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:30:43,873 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:30:43,873 :: INFO :: 8 cogs are loaded
2021-10-19 15:30:43,873 :: INFO :: -----------
2021-10-19 15:30:43,873 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:30:43,873 :: INFO :: dOGbot JSON API listening
2021-10-19 15:30:45,962 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:45:43,907 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:47:02,358 :: INFO :: dOGbot JSON API is shutting down...
2021-10-19 15:47:02,882 :: DEBUG :: > jishaku loaded!
2021-10-19 15:47:02,892 :: DEBUG :: > cogs.error_handling loaded!
2021-10-19 15:47:02,892 :: DEBUG :: > cogs.background_loop loaded!
2021-10-19 15:47:02,893 :: DEBUG :: > cogs.support_server_commands loaded!
2021-10-19 15:47:02,893 :: DEBUG :: > cogs.database_commands loaded!
2021-10-19 15:47:02,894 :: DEBUG :: > cogs.utility loaded!
2021-10-19 15:47:02,905 :: DEBUG :: > cogs.rest_api loaded!
2021-10-19 15:47:02,905 :: DEBUG :: > cogs.fun loaded!
2021-10-19 15:47:05,512 :: INFO :: -----------
2021-10-19 15:47:05,512 :: INFO :: The bot is ready.
2021-10-19 15:47:05,512 :: INFO :: Logged in as dOGbot (807690326700458005).
2021-10-19 15:47:05,512 :: INFO :: I see 3 guilds, and 1 members.
2021-10-19 15:47:05,513 :: INFO :: To invite your bot to your server, use the following link: https://discordapp.com/api/oauth2/authorize?client_id=807690326700458005&scope=bot&permissions=0
2021-10-19 15:47:05,513 :: INFO :: 8 cogs are loaded
2021-10-19 15:47:05,513 :: INFO :: -----------
2021-10-19 15:47:05,513 :: WARNING :: The status channel for the support server command is configured improperly.
2021-10-19 15:47:05,513 :: INFO :: dOGbot JSON API listening
2021-10-19 15:47:09,903 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -version
2021-10-19 15:49:09,521 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -help
2021-10-19 15:49:13,102 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -youtube
2021-10-19 15:49:17,398 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -twitch
2021-10-19 15:49:20,694 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -twitter
2021-10-19 15:49:47,338 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -kuki
2021-10-19 15:49:57,547 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -ping
2021-10-19 15:50:06,400 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -ping
2021-10-19 15:51:34,031 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -version
2021-10-19 15:54:37,204 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk reload cogs.utility
2021-10-19 15:54:44,049 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -credits
2021-10-19 15:55:41,302 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk reload cogs.utility
2021-10-19 15:55:47,316 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -version
2021-10-19 15:55:53,330 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -credits
2021-10-19 15:57:57,784 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk reload cogs.fun
2021-10-19 15:58:04,422 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -kuki
2021-10-19 15:58:12,397 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -kuki @dOGbone
2021-10-19 15:58:47,172 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -kuki @dOGbot
2021-10-19 15:58:59,487 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk reload cogs.fun
2021-10-19 15:59:04,071 :: INFO :: 772889538300608512 - #『🤖』cmds :: <dOGbone#0001> -kuki @hydrostaticcog for being so awesome <3
2021-10-19 15:59:06,525 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -kuki @dOGbot
2021-10-19 15:59:29,077 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk reload cogs.fun
2021-10-19 15:59:35,815 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -help
2021-10-19 15:59:55,103 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -kuki @dOGbone
2021-10-19 16:01:41,364 :: INFO :: 772889538300608512 - #『🤖』cmds :: <hydrostaticcog#4886> -jsk git commit -am "revamp :D"
